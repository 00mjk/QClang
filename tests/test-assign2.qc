int main() {
    tup(qubit, int) q;
    bit b;
    q = (true, 42);
    b = measure(q[0]);
    return 0;
}
