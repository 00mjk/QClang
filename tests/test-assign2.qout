COMPLETED
{'0 1 0 0': 1}
dict_keys(['0 1 0 0'])
