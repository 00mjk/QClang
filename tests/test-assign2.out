OPENQASM 2.0;
include "qelib1.inc";
qreg q[1];
creg c[1];
h q;
creg tmp0_bt0[1];
qreg q_0_qt0[1];
creg b_b0[1];
qreg temp_0[1];
x temp_0;
creg temp_0_mb1[1];
measure temp_0 -> temp_0_mb1;
